src/main/resources/swagger.api/schemas/instance.json

```diff
@@ -194,6 +194,7 @@
         "$ref": "holding.json"
       }
     }
-  }
+  },
+  "required": ["electronicAccess", "notes", "items", "holdings"]
```

Comment:
Why only these fields? There are much more arrays in the schema?
What will happen when an incoming record doesn't have these fields?

Reply:
Other fields are not a result of a join executed by Elasticsearch and therefore are not affected.